requirements.txt

```
@@ -26,9 +26,9 @@ uuid==1.30
 pytz==2017.2
 django-tables2==1.14.2
 tablib==0.12.1
-dj-database-url==0.4.2
+dj-database-url==0.5.0
 whitenoise==3.3.1
-psycopg2==2.7.1
+psycopg2>=2.7.2
```

I would prefer fixing it to 2.7.2 (or 2.7.6) if possible. Even though there's no major release, Django version is fixed in this project (basically bc it's a long process of migration to the newest release) and I don't know if psycopg2 may release a new version which may be uncompatible with previous editions.

```suggestion
psycopg2==2.7.2
```

---

Merging and changing on master. Not the best, but would like to have it fixed.